esp/src/src/ws_access.ts

```diff
@@ -307,12 +307,12 @@ var IndividualPermissionsStore = declare([Memory], {
 
     put: ESPUtil.override(function (inherited, row) {
         this.get(row.__hpcc_id);
-        var retVal = inherited();
+        var retVal = inherited(arguments);
         var request = {
             basedn: row.basedn,
             rtype: row.rtype,
             rtitle: row.rtitle,
-            rname: row.name,
+            rname: row.rname,
```

Comment:
Is this change deliberate?  Does it relate to a diff issue (if so reference it here).

Reply:
@GordonSmith, rolled back change.